distributed/worker.py

```diff
@@ -326,6 +326,11 @@ def _register_with_scheduler(self):
                 middle = (_start + _end) / 2
                 self.scheduler_delay = response['time'] - middle
                 self.status = 'running'
+
+                # Retrieve eventual init functions and run them
+                for init_function in pickle.loads(response['init-functions']):
+                    result = init_function()
```

Comment:
Could we just not make the signature `callable(dask_worker: Worker) -> None` in all cases? It doesn't seem to me to be much of a burden to the writer of an initialization function to mandate that it takes the worker instance as an argument whether or not the function actually uses it.

Reply:
> There is also a pre-existing convention for this in client.run which has proven useful.

I don't see where it is located. It could help me do implement this part for already connected workers.

Reply:
Search for `dask_worker` in `worker.py`.  It's under the `def run(...)` function.

Reply:
Thanks, was not looking into the correct run() method...

Reply:
Even if I put the signature with dask_worker in the docstring, I kept @mrocklin suggestion to be robbust to simple case where the function does'nt declare dask_worker.